### PR TITLE
Fix formatting for create custom domains with kn domain

### LINKED ISSUE #4225: Clean up PR #4209
Edit https://github.com/knative/docs/pull/4209 for style and fix formatting. Ensure the text follows the [Knative Style Guide](https://knative.dev/development/help/contributor/#style).

### PR DESCRIPTION
**Pull request by snneji**
Fixes #4225

Fixed the formatting issues and made a few other tweaks.

### CODE REVIEW

**Comment by omerbensaadon** [comment]
/approve

Would it be worth creating a "Serving Prerequisites Snippet," is this something we want on many pages?

File: docs/developer/serving/services/custom-domains.md [hunk 1]
```diff
@@ -48,62 +48,74 @@ DomainMappings in that namespace to use the domain name.
             ```
             Where `<filename>` is the name of the file you created in the previous step.
 
-    * To create ClusterDomainClaims automatically, set the `autocreateClusterDomainClaims` property
+    * To create ClusterDomainClaims automatically: set the `autocreateClusterDomainClaims` property
```
Comment (abrennan89) [inline docs/developer/serving/services/custom-domains.md:51]: I think it was better with a comma, not sure why use a `:` here?

File: docs/developer/serving/services/custom-domains.md [hunk 2]
```diff
@@ -48,62 +48,74 @@ DomainMappings in that namespace to use the domain name.
...
-    Where:
-    - `<domain-name>` is the domain name that you want to map a Service or Route to.
-    - `<target>` is the name of the Service or Route that is mapped to the domain. You can use a prefix `ksvc:` or `kroute:` to specify whether to map the domain to a Knative Service or Route. If no prefix is given, `ksvc:` is assumed. Additionally, you can use a `:namespace` suffix to point to a Service or Route in a different namespace than this Domain mapping. E.g. `mysvc` maps to a Service `mysvc` in the same namespace as this mapping, while `kroute:myroute:othernamespace` maps to a Route `myroute` in namespace `othernamespace`.
-    - `<tls-secret>` is optional and if provided enables the TLS protocol. The value specifies the secret that holds the server certificate.
-    - `<namespace>` is the namespace where the DomainMapping  should be created. By default the DomainMapping is created in the current namespace.
+    === "kn"
+        Run the command:
+
+        ```bash
+        kn domain create <domain-name> --ref <target> --tls <tls-secret> --namespace <namespace>
```
Comment (abrennan89) [inline docs/developer/serving/services/custom-domains.md:99]:
```suggestion
        kn domain create <domain-name> --ref <target> --tls <tls-secret> -n <namespace>
```

Comment (abrennan89) [reply docs/developer/serving/services/custom-domains.md:99]: I'd check with eng if this is OK but I think we said a while ago we'd shorten these for consistency?

**Review by abrennan89 — APPROVED** [review]
couple of minor comments, otherwise lgtm

**Pull request merged**